packages/components/molecules/f-mega-modal/README.md

```diff
@@ -30,7 +27,7 @@ npm install @justeat/f-mega-modal
 
 ### Vue Applications
 
-You can import it in your Vue SFC like this (please note that styles have to be imported separately):
+> Styles have to be imported separately.
```

Comment:
I would change this to be: `Please note that styles have to be imported separately to the rest of the component.`

Reply:
I think that's strongly implied by the structure.
I removed all the verbose/chatty language so that there's less text to compare and keep synced.

For example:  _Please not that_ is redundant as we wouldn't have written it if we didn't want the reader to note it and _to the rest of the component_ is redundant as the component is explicitly the subject of the document.

I think that unless some important information has been lost then we should keep the word count to a minimum.

Reply:
I kind of see the `Please note` as bringing special attention to the fact that it's separate to the component import (as when explaining the component setup to people, this isn't always obvious). Just reads a little better to me like that.